pkg/storage/cacher.go

```diff
@@ -752,7 +752,7 @@ func (c *cacheWatcher) process(initEvents []watchCacheEvent, resourceVersion uin
 	const initProcessThreshold = 500 * time.Millisecond
 	startTime := time.Now()
 	for _, event := range initEvents {
-		c.sendWatchCacheEvent(event)
+		c.sendWatchCacheEvent(&event)
```

Comment:
Yes (and this is true assumption today). I added explicit comment for that method.